improver/utilities/spatial.py

```diff
@@ -100,9 +100,9 @@ def calculate_grid_spacing(cube, units, axis='x'):
     return diffs[0]
 
 
-def convert_distance_into_number_of_grid_cells(
+def distance_to_number_of_grid_cells(
```

Comment:
docstrings might need some work?
This function doesn't require an equal-area grid, thats why it can take an axis?